upload/src/addons/SV/NoticeTimeReplacable/_files/js/sv/notice-time-replacable/core.js

```diff
@@ -0,0 +1,131 @@
+var SV = window.SV || {};
+SV.NoticeTimeReplacable = SV.NoticeTimeReplacable || {};
+
+!function($, window, document, _undefined)
+{
+    "use strict";
+
+    SV.NoticeTimeReplacable.RelativeTimestamp = XF.Element.newHandler({
+        options: {
+            timestamp: null,
+        },
+
+        timer: null,
+
+        init: function()
+        {
+            if (!this.options.timestamp)
+            {
+                console.error('Timestamp is missing.');
+                return;
+            }
+
+            this.timer = setInterval(XF.proxy(this, 'updateTime'), 1000);
+        },
+
+        /**
+         * @param {Number|String} value
+         * @param {String} phrase
+         */
+        getDatePart: function (value, phrase)
+        {
+            if (typeof value === 'string')
+            {
+                value = parseInt(value) || 0;
+            }
+
+            if (typeof phrase !== 'string' || !phrase)
+            {
+                console.error('Invalid phrase provided.');
+                clearInterval(this.timer);
```

Comment:
The other `return` doesn't trigger this, is this expected?

Reply:
Should return now.